nomt/src/store/sync.rs

```diff
@@ -58,19 +64,19 @@ impl Sync {
             page_diffs,
         );
 
-        let (beatree_trigger_fsync_rx, meta_wd) =
-            spawn_prepare_sync_beatree(&self.tp, &mut value_tx, beatree.clone());
-
-        let (bbn_writeout_done, ln_writeout_done) = spawn_fsync_beatree(
+        let meta_wd = spawn_prepare_sync_beatree(
```

Comment:
I find `meta_wd` confusing, it makes me think that `spawn_prepare_sync_beatree` does something other than just creating `beatree::SyncData`. Maybe something like `beatree_wd` could be clearer. Because the meta file is created later in the same function

Reply:
Names and logic have been updated in #576 there it's definitely clearer. There is a concept of a manifest (meta), and the beatree produces something that needs to be stored in it